pkg/imgutil/dockerconfigresolver/credentialsstore_test.go

```diff
@@ -169,6 +169,7 @@ func TestBrokenCredentialsStore(t *testing.T) {
 	}
 
 	t.Run("Broken Docker Config testing", func(t *testing.T) {
```

Comment:
there is also a healthy Docker configuration

Reply:
Sure. Suggesting we clean that up at a later point?

Reply:
sure